examples/braket_features/Verbatim_Compilation.ipynb

```diff
@@ -792,6 +792,132 @@
     "print(result.measurement_counts)"
```

Comment:
<p>With this information, you can write a circuit to run verbatim on the IonQ device. IonQ currently only supports verbatim compilation for the entire circuit, so every instruction in the circuit will need to be enclosed in a verbatim box. In other words, you cannot have any gates outside of the verbatim box. As well, note that IonQ native gates cannot be used outside of a verbatim box. To learn more about IonQ native gates and best practices for using them, see the <a href="https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html?tag=local002-20#braket-qpu-partner-ionq" target="_blank">Amazon Braket Developer Guide</a> and <a href="https://ionq.com/docs/getting-started-with-native-gates" target="_blank">IonQ native gate documentation</a>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>